domain-management/src/main/java/org/jboss/as/domain/management/security/UserPropertiesFileLoader.java

```diff
@@ -101,7 +101,7 @@ protected void load() throws IOException {
         try {
             br = Files.newBufferedReader(propertiesFile.toPath(), StandardCharsets.UTF_8);
             String currentLine;
-            while (realmName == null && (currentLine = br.readLine()) != null) {
+            while ((currentLine = br.readLine()) != null) {
```

Comment:
With this it's now possible that more than one realm comment will be parsed. That should probably be handled, either with an error or by ignoring subsequent realm comments.